innovation.scss

```diff
@@ -0,0 +1,3059 @@
+			top: 25px;
+			left: 30px;
+		}
+		&.bottom_left_icon {
+			bottom: 23px;
+			left: 30px;
+		}
+		&.bottom_center_icon {
+			bottom: 23px;
+			left: 195px;
+		}
+		&.bottom_right_icon {
+			bottom: 23px;
+			right: 26px;
+		}
+	}
+}
+
+/* Hexagon icons take up ~27 % of a real cards width. 
+*  Our cards will shoot for 25 % ->  M.45px
+*  If you ever change the background-size, you must edit the offset in css_generator.py
+*/
+.hexagon_card_icon {
+	background-image: url("img/hexagon_icons.png");
+	position: absolute;
+	&.M {
+		background-size: 735px, auto;
+		width: 45px;
+		height: 45px;
+		&.top_left_icon {
+			top: 2px;
+			left: 4px;
+		}
+		&.bottom_left_icon {
+			bottom: 1px;
+			left: 4px;
+		}
+		&.bottom_center_icon {
+			bottom: 1px;
+			left: 69px;
+		}
+		&.bottom_right_icon {
+			bottom: 1px;
+			right: 2px;
+		}
+	}
+	&.L {
+		background-size: 1511px, auto;
+		width: 90px;
+		height: 90px;
+		&.top_left_icon {
+			top: 17px;
+			left: 26px;
+		}
+		&.bottom_left_icon {
+			bottom: 11px;
+			left: 26px;
+		}
+		&.bottom_center_icon {
+			bottom: 11px;
+			left: 192px;
+		}
+		&.bottom_right_icon {
+			bottom: 11px;
+			right: 15px;
+		}
+	}
+}
+
+.recto {
+	display: inline-block;
+	position: relative;
+	border: solid 1px #ffffff;
+	box-shadow: -1px 1px grey;
+	border-radius: 3px;
+	&.S{
+		width: 31px;
+		height: 45px;
+		&.age_1,&.age_2,&.age_3,&.age_4,&.age_5,&.age_6,&.age_7,&.age_8,&.age_9,&.age_10{
+			background-image: url("img/card_backs_portrait.jpg");
+			background-size: 380px 256px;
+		}
+		&.age_1 {
+			background-position-x: -2px;
+		}
+		&.age_2 {
+			background-position-x: -36px;
+		}
+		&.age_3 {
+			background-position-x: -71px;
+		}
+		&.age_4 {
+			background-position-x: -105px;
+		}
+		&.age_5 {
+			background-position-x: -140px;
+		}
+		&.age_6 {
+			background-position-x: -174px;
+		}
+		&.age_7 {
+			background-position-x: -209px;
+		}
+		&.age_8 {
+			background-position-x: -243px;
+		}
+		&.age_9 {
+			background-position-x: -278px;
+		}
+		&.age_10 {
+			background-position-x: -312px;
+		}
+		&.type_0 {
+			background-position-y: -3px;
+		}
+		&.type_1 {
+			background-position-y: -54px;
+		}
+		&.type_2 {
+			background-position-y: -105px;
+		}
+		&.type_3 {
+			background-position-y: -156px;
+		}
+		&.type_4 {
+			background-position-y: -208px;
+		}
+		&.relic {
+			border: solid 1px transparent;
+			background-position-x: -347px;
+		}
+		&.age_null{
+			background-image: url("img/special_achievements_portrait.jpg");
+			background-size: 175px, auto;
+		}
+		&.item_105 {
+			background-position: -1px -1px;
+		}
+		&.item_106 {
+			background-position: -36px -1px;
+		}
+		&.item_107 {
+			background-position: -71px -1px;
+		}
+		&.item_108 {
+			background-position: -106px -1px;
+		}
+		&.item_109 {
+			background-position: -141px -1px;
+		}
+	}
+}
+/* scss_generator code starts here */
+/* anything below this will get blown away on rebuild */
+
+.font_size_1 {
+	font-size: 0.0625rem;
+}
+
```

Comment:
Can't we do something like this here? https://stackoverflow.com/questions/50699726/calculation-based-on-class-name-in-sass

I expect we can also do something similar for the hexagons. With SCSS, might we be able to eliminate the Python script altogether?

Reply:
OOO yes we can, I'll look to do that